### PR TITLE
Use develop version of php compatibility check tool, for 8.0 and 8.1 checks

### PR DESCRIPTION
**Pull request by kierenevans**
Ref: https://github.com/PHPCompatibility/PHPCompatibility/issues/1312

The last release was in 2019 for php 7.4.

### CODE REVIEW

File: src/akeneo/application/skeleton/composer.json.twig [hunk 1]
```diff
@@ -102,9 +102,14 @@
...
+    "allow-plugins": {
+      "dealerdirect/phpcodesniffer-composer-installer": true,
+      "symfony/flex": true
+    },
     "platform": {
       "php": "{{ (version_compare(@('akeneo.major_version'), '5', '>=')) ? '7.4' : '7.3.22' }}"
     }
   },
-  "minimum-stability": "stable"
+  "minimum-stability": "alpha",
```
Comment (andytson-inviqa) [inline src/akeneo/application/skeleton/composer.json.twig]: if you need alpha packages, put the `@alpha` on the package, don't allow all packages to be alpha

**Review by andytson-inviqa — CHANGES_REQUESTED** [review]
minimum-stability should remain stable

File: src/wordpress/application/skeleton/composer.json.twig [hunk 2]
```diff
@@ -12,7 +12,8 @@
         "behat/mink-goutte-driver": "^1.2",
         "dmore/behat-chrome-extension": "^1.3",
         "dmore/chrome-mink-driver": "^2.7",
-        "phpcompatibility/php-compatibility": "dev-master",
+        "phpcompatibility/php-compatibility": "dev-develop@alpha",
+        "phpcsstandards/phpcsutils": "1.0.0-alpha4",
```
Comment (andytson) [inline src/wordpress/application/skeleton/composer.json.twig:16]: https://packagist.org/packages/phpcsstandards/phpcsdevtools#1.0.0 there's a release out now

Comment (kierenevans) [reply src/wordpress/application/skeleton/composer.json.twig:16]: Different package unfortunately :(

**Review by andytson-inviqa — DISMISSED** [review]

**Review by andytson-inviqa — DISMISSED** [review]

**Comment by andytson-inviqa** [comment]
```
Root composer.json requires phpcompatibility/php-compatibility dev-develop@alpha, found phpcompatibility/php-compatibility[dev-develop, 10.x-dev (alias of dev-develop)] but it does not match your minimum-stability.
```
ugh, an evil to solve an lesser evil

I don't get it

```
(Note that you can also specify stability requirements on a per-package basis using stability flags in the version constraints that you specify in a require block (see [package links](https://getcomposer.org/doc/04-schema.md#package-links) for more details).
```

https://github.com/inviqa/harness-base-php/blob/1.5.x/src/drupal8/application/skeleton/composer.json.twig#L6
https://github.com/inviqa/harness-base-php/blob/1.5.x/src/drupal8/application/skeleton/composer.json.twig#L57

fine there

**Comment by andytson-inviqa** [comment]
Does it need `@alpha` for dev-develop? `@dev` is probably the correct one, but I think is implied automatically

`https://github.com/PHPCompatibility/PHPCompatibility/issues/1306`

**Comment by andytson-inviqa** [comment]
possibly just a magento1 error now:

```
#11 54.26 Your requirements could not be resolved to an installable set of packages.
#11 54.26 
#11 54.26   Problem 1
#11 54.26     - Root composer.json requires phpcompatibility/php-compatibility dev-develop -> satisfiable by phpcompatibility/php-compatibility[dev-develop].
#11 54.26     - phpcompatibility/php-compatibility dev-develop requires squizlabs/php_codesniffer ^3.7.1 -> found squizlabs/php_codesniffer[3.7.1] but it conflicts with your root composer.json require (3.2.2).
#11 54.26   Problem 2
#11 54.26     - Root composer.json requires phpcsstandards/phpcsutils 1.0.0-alpha4 -> satisfiable by phpcsstandards/phpcsutils[1.0.0-alpha4].
#11 54.26     - phpcsstandards/phpcsutils 1.0.0-alpha4 requires squizlabs/php_codesniffer ^3.7.1 || 4.0.x-dev@dev -> found squizlabs/php_codesniffer[3.7.1] but it conflicts with your root composer.json require (3.2.2).
```

**Review by andytson-inviqa — APPROVED** [review]

**Pull request merged**